### PR TITLE
ignore missing filelists in metadata (issue #273)

### PR DESCRIPTION
**Pull request by oliverkurth**
closes issue #273 .

### CODE REVIEW

**Review by tapakund — APPROVED** [review]

File: pytests/tests/test_repo_missing_metadata.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+# of the License are located in the COPYING file of this distribution.
+#
+#   Author: Oliver Kurth <okurth@vmware.com>
+
+import os
+import shutil
+import pytest
+import errno
+
+REPODIR='/root/repofrompath/yum.repos.d'
```
Comment (sshedi) [inline pytests/tests/test_repo_missing_metadata.py]: Use `REPODIR=/tmp/...` if test fails, our root directory won't contain these files

File: pytests/tests/test_repo_missing_metadata.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+REPOFILENAME='repofrompath.repo'
+REPONAME="repofrompath-test"
+WORKDIR='/root/repofrompath/workdir'
```
Comment (sshedi) [inline pytests/tests/test_repo_missing_metadata.py]: same here

**Review by sshedi — APPROVED** [review]
Minor nit picking comment. LGTM

File: pytests/tests/test_repo_missing_metadata.py (hunk 1 above)
Comment (oliverkurth) [reply pytests/tests/test_repo_missing_metadata.py]: Should be okay since we clean it up, but I changed it anyway.

**Pull request merged**